pkg/controller/util.go

```diff
@@ -721,3 +729,20 @@ func MakeUploadServiceSpec(name string, pvc *v1.PersistentVolumeClaim) *v1.Servi
 	}
 	return service
 }
+
```

Comment:
At some point I'd like to clean up util.go, probably have a pod.go file for all the pod related stuff and share it across the controllers, but for now util is a nice dumping ground for this sort of thing.